lib/kernel/test/interactive_shell_SUITE.erl

```diff
@@ -1407,6 +1409,26 @@ shell_help(Config) ->
         stop_tty(Term),
         ok
     end.
+
+prompt_2(_) ->
+    ["\e]94m",54620,44397,50612,47,51312,49440,47568,"\e]0m"].
+shell_escape_sequence_end_of_prompt_followed_by_unicode(_Config) ->
+    %% If the prompt ended with an escape sequence, and the user input a unicode character
+    %% prim_tty tried to put that character in the escape sequence. This test checks that
+    %% the unicode character will not be part of the escape sequence.
+    Prompt = prompt_2(none),
+    rtnode:run(
+  [{putline,"÷."},
+            {expect, "÷[.]"},
+            {expect, ["\\Q",Prompt,"\\E"]}],
+            "", "",
+            ["-pz",filename:dirname(code:which(?MODULE)),
+            "-connect_all","false",
+            "-kernel","logger_level","all",
+            "-kernel","shell_history","disabled",
+            "-kernel","prevent_overlapping_partitions","false",
+            "-eval","shell:prompt_func({interactive_shell_SUITE,prompt_2})."
```

Comment:
```suggestion
```

The default prompt is `["\e[94m",54620,44397,50612,47,51312,49440,47568,"\e[0m"]`, so no need to do this.

Reply:
It's a minor difference, the default prompt is: `io_lib:format("(~ts)~w> ",[Fun(),N])`. i.e. `[$(,"\e[94m",54620,44397,50612,47,51312,49440,47568,"\e[0m",$),$1,$>,$ ]` This above tests that the escape character can be the last character of the prompt.